python/heterocl/api.py

```diff
@@ -306,6 +307,18 @@ def build(schedule, target=None, name="default_function", stmt=None):
             new_inputs.append(i.tensor.op.output(0))
         else:
             new_inputs.append(i.var)
+
+    # auto data moving to dev
+    if len(schedule.placement) == 0 and (target is not None):
+        if not isinstance(target, str):
+            import builtins as __builtin__
+            __builtin__.print("[HCL] Auto data palcement...")
```

Comment:
Comment out this for now and add a TODO. We will be using a better message system later.